thumbor/thumbor.conf

```diff
@@ -1,260 +1,221 @@
+## MALICIOUS USERS
 ALLOW_UNSAFE_URL = True
 
-# Mixed storage classes. Change them to the fullname of the
-# storage you desire for each operation.
-#MIXED_STORAGE_FILE_STORAGE = 'thumbor.storages.file_storage'
-#MIXED_STORAGE_CRYPTO_STORAGE = 'thumbor.storages.no_storage'
-#MIXED_STORAGE_DETECTOR_STORAGE = 'thumbor.storages.no_storage'
-
-
-# This configuration indicates whether thumbor should use a
-# custom error handler. 
-# Defaults to: False
-#USE_CUSTOM_ERROR_HANDLING=False
-#ERROR_HANDLER_MODULE='thumbor.error_handlers.sentry'
-#SENTRY_DSN_URL=''
-#SENTRY_ENVIRONMENT=None
-
-#FILTERS = [
-    #'thumbor.filters.brightness',
-    #'thumbor.filters.colorize',
-    #'thumbor.filters.contrast',
-    #'thumbor.filters.rgb',
-    #'thumbor.filters.round_corner',
-    #'thumbor.filters.quality',
-    #'thumbor.filters.noise',
-    #'thumbor.filters.watermark',
-    #'thumbor.filters.equalize',
-    #'thumbor.filters.fill',
-    #'thumbor.filters.saturation',
-    #'thumbor.filters.sharpen',
-    #'thumbor.filters.strip_exif',
-    #'thumbor.filters.strip_icc',
-    #'thumbor.filters.frame',
-    #'thumbor.filters.grayscale',
-    #'thumbor.filters.rotate',
-    #'thumbor.filters.format',
-    #'thumbor.filters.max_bytes',
-    #'thumbor.filters.no_upscale',
-
-    ## can only be applied if there are already points for the image being served
-    ## this means that either you are using the local face detector or the image
-    ## has already went through remote detection
-    ## 'thumbor.filters.redeye',
-#]
-
-#OPTIMIZERS = [
-    #'thumbor.optimizers.jpegtran',
-    #'thumbor.optimizers.gifv',
-#]
+## Mixed storage classes. Change them to the fullname of the storage you desire
+## for each operation.
+# MIXED_STORAGE_FILE_STORAGE = "thumbor.storages.file_storage"
+# MIXED_STORAGE_CRYPTO_STORAGE = "thumbor.storages.no_storage"
+# MIXED_STORAGE_DETECTOR_STORAGE = "thumbor.storages.no_storage"
+
+
+## This configuration indicates whether thumbor should use a custom error
+## handler. Defaults to: False
+# USE_CUSTOM_ERROR_HANDLING=False
+# ERROR_HANDLER_MODULE="thumbor.error_handlers.sentry"
+# SENTRY_DSN_URL=""
+# SENTRY_ENVIRONMENT=None
+
+FILTERS = BUILTIN_FILTERS
+
+## Can only be applied if there are already points for the image being served
+## this means that either you are using the local face detector or the image
+## has already went through remote detection
+# FILTERS = BUILTIN_FILTERS + [
+#     "thumbor.filters.redeye",
+# ]
+
+# OPTIMIZERS = [
+#     "thumbor.optimizers.jpegtran",
+#     "thumbor.optimizers.gifv",
+# ]
 
 # from thumbor.handler_lists import BUILTIN_HANDLERS
 
 # HANDLER_LISTS = BUILTIN_HANDLERS + [
-#   # Add your handler list modules here
+#    # Add your handler list modules here
 # ]
 
-#JPEGTRAN_PATH = '/usr/local/bin/jpegtran'
+# JPEGTRAN_PATH = "/usr/local/bin/jpegtran"
 
-# If the CurlAsyncHTTPClient should be used
-# Defaults to: False
+## If the CurlAsyncHTTPClient should be used Defaults to: False
```

Comment:
For a quick read, defaults are now much harder to read. Please consider using a newline for them.